crates/api_common/src/build_response.rs

```diff
@@ -91,16 +90,19 @@ pub async fn build_post_response(
 #[tracing::instrument(skip_all)]
 pub async fn send_local_notifs(
   mentions: Vec<MentionData>,
-  comment: &Comment,
+  comment_id: CommentId,
```

Comment:
Rather than pre-fetch and provide the comment, post, and community, I realized it'd faster to just have the comment_id here, and below do a single `CommentView` fetch to get all the necessary info.